packages/frontend/src/components/Explorer/CustomMetricModal/index.tsx

```diff
@@ -323,7 +323,7 @@ export const CustomMetricModal = () => {
                             <Accordion.Panel>
                                 <FiltersProvider
                                     projectUuid={projectUuid}
-                                    fieldsMap={fieldsMap}
+                                    itemsMap={fieldsMap}
```

Comment:
perhaps the filtering in: https://github.com/lightdash/lightdash/pull/10062/files#r1599738715 should be done before here? 
```
itemsMap={dimensionsMap}
```
?

Reply:
ideally yes

Reply:
done here: https://github.com/lightdash/lightdash/pull/10062/commits/6ec4497e9149073f79ff1042af5a0b6aa6e14d46